src/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -147,18 +147,6 @@ public boolean match(final Territory t) {
     };
   }
 
-  public static Match<Territory> territoryCanMoveLandUnitsAndIsEnemy(final PlayerID player, final GameData data) {
```

Comment:
@ron-murhammer: again, this is your baby, so please decide which of these, if any, you want to keep